vllm/distributed/parallel_state.py

```diff
@@ -45,8 +45,8 @@ class GraphCaptureContext:
 
 
 def _split_tensor_dict(
-        tensor_dict: Dict[str, Union[torch.Tensor, Any]],
-        prefix: str = "") -> Tuple[List[Tuple[str, Any]], List[torch.Tensor]]:
+    tensor_dict: Dict[Any, Union[torch.Tensor, Any]]
```

Comment:
`tensor_dict: Dict[Any, Union[torch.Tensor, Any]]` --> `tensor_dict: Dict[str, Union[torch.Tensor, Any]]` ?

the key of dict should be str